pykerb/kerberos.c

```diff
@@ -458,11 +474,26 @@ static PyMethodDef KerberosMethods[] = {
     {NULL, NULL, 0, NULL}        /* Sentinel */
 };
 
-PyMODINIT_FUNC initkerberos(void)
+static struct PyModuleDef moduledef = {
+        PyModuleDef_HEAD_INIT,
```

Comment:
I would suggest referencing the field names explicitly here (helps to prevent silly mistakes should anything change in the structure):
`.m_name = PyModuleDef_HEAD_INIT,`
`.m_name = "Kerberos"`

and so on.

Reply:
Done